### ISSUE TITLE
Layout shifts to the right when categories nav button is clicked

### ISSUE THREAD

**Issue by sumairq** [prezly/theme-nextjs-bea#1139]
# Description
When the `categories` nav button is clicked, the layout shifts to the right. As content that exceeds the viewport gets hidden and the layout undergoes unintended shifts. This occurs because the scroll bar, which is removed due to the `overflow: hidden;` setting, causes the layout to shift to the right, resulting in a displacement of content.

## Steps to Reproduce

1. Click the `categories` navigation button.
2. Observe the layout shift as the vertical scroll bar is removed, leading to a displacement of content to the right.

![layout-shift](https://github.com/prezly/theme-nextjs-bea/assets/60618877/33b56425-610f-48e5-8ac2-2626b8919218)

## Expected Behavior
The layout should remain stable when the `categories` button is clicked, without any shifts caused by the absence of the scroll bar.

## Actual Behavior:
The layout shifts to the right when the categories dropdown is clicked due to the removal of the scroll bar, causing content displacement and potentially affecting user experience and visual consistency.

## Environment

Operating System: [Ubuntu 22.04.3 LTS]
Browser: [Chrome]

**Comment by MohammadxAli** [comment]
Hi @sumairq 👋

Thanks for reporting this issue 👍

Sure, feel free to open a PR.

**Comment by sumairq** [comment]
Hi @MohammadxAli 

Let me clarify the complete picture about what happened. 

First of all my account never got compromised before. It's the only time. Call it a lapse of judgement on my part.
Secondly I have reinstalled my operating system to ensure nothing malicious remains within my system. 

The problem happened when a mysterious "recruiter" tried to reach out to me through linkedIn . They appeared to be legit. They shared a figma design with me and had a website. Although I must say I did miss some red flags. 

That fake recruiter wanted me to clone their repo as a pretext for preparing me for the supposed "interview". As soon as I cloned the repository and installed the packages to run the project in my local environment. I recieved automated emails from Github that some activity is happening in my github account. Someone changed the email from "sumairqaisar992@gmail.com" my email to "smithdylan0919@gmail.com". All of sudden I lost access to my account. I thought it promt to convey the message wherever I was collaborating through my newly created account (totally uncompromised and safe).

Luckily they changed the username so I was able to create a fresh account with my old username.

I did some research, and found out that apparently the hacker did session hijacking on my account by getting me to open malicious file/script that he then used to dump my session and move it to their browser.

![image](https://github.com/prezly/theme-nextjs-bea/assets/170146330/bb6670d9-717c-4d08-bb3a-06c74be80774)

It is my humble request to you to not punish me for something I was a victim of.
I have made sure that nothing malicious remains within my system or browser by reinstalling my ubuntu operating system and have resolved to not clone or run any project locally unless I am 100% certain of the company's legitimacy.

### Here is my LinkedIn 
https://www.linkedin.com/in/sumairq/

Here is link to LinkedIn post where I explained what happened and alerted my network about the scam.
https://www.linkedin.com/posts/sumairq_cybersecurity-github-phishing-activity-7197343472301813760-A1DO?utm_source=share&utm_medium=member_desktop

I can provide a reference from the team I was working with at Mozilla as an open-source developer in an attempt to restore trust. 
I have graduated from Microverse Coding bootcamp and can get references from them as well.
Or any other way to prove that I am a legitimate developer aiming to contribute to prezly and by no means attempting to compromise its code.

Regards,
Sumair

**Comment by sumairq** [comment]
Hi @MohammadxAli @e1himself @kudlajz

Great News ! 

I can confirm that thanks to Github Support :raised_hands:  , I have been able to regain access to my original account. 
I was able to restore my original name and username to that account as well.
Secured it with 2 Factor authentication, changed the password.

I want to continue contributing to Prezly. I hope you will allow me a chance to contribute . Your positive response in this regard will mean alot to me. 

Thank you again for bearing with me during this period.
 
 Regards,
 Sumair Qaisar
